app/models/concerns/common_metadata.rb

```diff
@@ -19,6 +19,9 @@ module CommonMetadata
     property :state, predicate: ::F3Access.objState, multiple: false do |index|
       index.as :stored_searchable, :facetable
     end
+    property :workflow_note, predicate: ::RDF::Vocab::MADS.note do |index|
```

Comment:
:shudder:. Can we find anything better than MODS (or MADS)? I can have a look around in a few.